scripts/tekton/README.md

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
Can we re-write this whole pipeline / doc to be generic.

Reply:
Possibly, it was what Jarek wrote. Will take a look.

Reply:
Removed